llvm/include/llvm/ProfileData/MemProfReader.h

```diff
@@ -213,6 +213,19 @@ class RawMemProfReader final : public MemProfReader {
 class YAMLMemProfReader final : public MemProfReader {
 public:
   YAMLMemProfReader() = default;
+
+  // Return true if the \p DataBuffer starts "---" indicating it is a YAML file.
+  static bool hasFormat(const MemoryBuffer &DataBuffer);
+  // Return true if the file at \p Path starts with magic bytes indicating it is
+  // a raw binary memprof profile.
```

Comment:
Update this comment to indicate that it's just a wrapper around the other one which reads the file as a memorybuffer?

Reply:
Done.